_data/finance.yml

```diff
@@ -193,6 +206,7 @@ websites:
     - name: Shoeboxed
       url: https://www.shoeboxed.com/
       twitter: Shoeboxed
+      facebook: shoeboxed
```

Comment:
i think this, like the `spotify` etc. stuff, is a result of facebook serving pages based off of some sort of geographic information (IP address, facebook profile, etc)